packages/aws-cdk-lib/aws-eks/lib/cluster.ts

```diff
@@ -1817,11 +1818,13 @@ export class Cluster extends ClusterBase {
    * @param options options for creating a new nodegroup
    */
   public addNodegroupCapacity(id: string, options?: NodegroupOptions): Nodegroup {
-    const hasInferentiaInstanceType = [
+    const hasInferentiaOrTraniumInstanceType = [
```

Comment:
could we correct this small nit for `hasInferentiaOrTraniumInstanceType` to `hasInferentiaOrTraniumInstanceType`